src/mame/drivers/spectrum.cpp

```diff
@@ -499,15 +499,15 @@ Number Keys are the only keys not having a function in BASIC Mode (hence, they o
 in EXT+Shift Mode on a real Spectrum).
 */
 
-/* TO DO: replace PORT_CHAR('\xD7') with an 'empty' PORT_CHAR. I used \xD7 (multiplication sign) just as a placeholder. There should be no
+/* TO DO: replace PORT_CHAR(0xD7) with an 'empty' PORT_CHAR. I used \xD7 (multiplication sign) just as a placeholder. There should be no
```

Comment:
It might be a good idea to map the alternate functions of the number row to the F-keys in natural keyboard mode.  Probably better than making them all respond to multiplication sign.